go.mod

```diff
@@ -41,17 +42,17 @@ require (
 	github.com/gorilla/mux v1.8.1
 	github.com/rancher/dynamiclistener v0.6.0-rc2
 	github.com/rancher/lasso v0.0.0-20240705194423-b2a060d103c1
-	github.com/rancher/rancher/pkg/apis v0.0.0-20240628084651-0dd62c26260a
-	github.com/rancher/rke v1.6.0-rc7
+	github.com/rancher/rancher/pkg/apis v0.0.0-20240710151157-3a6de11e4de9
+	github.com/rancher/rke v1.6.0-rc9
 	github.com/rancher/wrangler/v3 v3.0.0
 	github.com/robfig/cron v1.2.0
 	github.com/sirupsen/logrus v1.9.3
 	github.com/stretchr/testify v1.9.0
 	golang.org/x/exp v0.0.0-20240416160154-fe59bbe5cc7f
 	golang.org/x/text v0.16.0
 	golang.org/x/tools v0.21.1-0.20240508182429-e35e4ccd0d2d
-	k8s.io/api v0.30.1
-	k8s.io/apimachinery v0.30.1
+	k8s.io/api v0.30.2
+	k8s.io/apimachinery v0.30.2
```

Comment:
Can you elaborate on why this was bumped? If this is getting replaced above I'm not sure that I see the value.

Reply:
I did not update this manually but this was automatically picked from rancher's require section with `go mod tidy`. Not sure why it ignores the `replace` directive.